base/sharedarray.jl

```diff
@@ -249,7 +249,7 @@ function rand!{T}(S::SharedArray{T})
 end
 
 function randn!(S::SharedArray)
-    f = S->map!(x->randn, S.loc_subarr_1d)
+    f = S->map!(x->randn(), S.loc_subarr_1d)
```

Comment:
Is there a reason this can't just be:

```julia
f = S->randn!(S.loc_subarr_1d)
```

(similarly for `rand!`)

cc @carlobaldassi